addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBridgeHandler.java

```diff
@@ -136,21 +136,26 @@ public void responseReceived(BaseResponse response) {
                                     public void responseReceived(BaseResponse response) {
 
                                         if (response.isOK()) {
+                                            logger.info("New Id successfully set!");
                                             updateState(channelUID, new StringType("New Id successfully set"));
                                         } else if (response.getResponseType() == ResponseType.RET_FLASH_HW_ERROR) {
+                                            logger.warn("The write/erase/verify process failed!");
                                             updateState(channelUID,
                                                     new StringType("The write/erase/verify process failed"));
                                         } else if (response.getResponseType() == ResponseType.RET_BASEID_OUT_OF_RANGE) {
+                                            logger.warn("Base id out of range!");
                                             updateState(channelUID, new StringType("Base id out of range"));
                                         } else if (response.getResponseType() == ResponseType.RET_BASEID_MAX_REACHED) {
+                                            logger.warn("No more change possible!");
                                             updateState(channelUID, new StringType("No more change possible"));
                                         }
                                     }
 
                                 });
 
-                    } catch (Exception e) {
-                        updateState(channelUID, new StringType("Id could not be parsed"));
+                    } catch (IllegalArgumentException e) {
+                        logger.warn("BaseId could not be parsed!", e);
```

Comment:
Please remove this logging. The channel already contains this message and the framework logs channel changes.

Reply:
OK, but see last comment about exception logging.